config/language_server_node_eslint.yml

```diff
@@ -3,7 +3,7 @@
 # Configuration for the JAVA LSP server component
 ---
 version: 0.0.+
-repository_root: "https://lsp-component-eslint.cfapps.eu10.hana.ondemand.com/"
+repository_root: "https://lsp-component-eslint.cfapps.sap.hana.ondemand.com/"                  
```

Comment:
You can't submit with move to canary landscape which is not available on factory

Reply:
fixed